include/openPMD/backend/Attribute.hpp

```diff
@@ -70,7 +71,7 @@ class Attribute :
                             bool >
 {
 public:
-    Attribute(resource r) : Variant(r)
+    Attribute(resource r) : Variant(std::move(r))
```

Comment:
note: single argument constructors (converting constructors) should also be `explicit` in clean code. but this breaks to much in our usage for now.
https://en.cppreference.com/w/cpp/language/converting_constructor